packages/leva/src/components/Select/select-plugin.ts

```diff
@@ -10,7 +10,7 @@ export const schema = (_o: any, s: any) =>
     .test(s)
 
 export const format = (value: any, { values }: InternalSelectSettings) => {
-  return values.indexOf(value)
+  return value
```

Comment:
When not explicitly set by an input the format function returns the value, so we might as well remove the whole function :)